src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    private final Map<String, Integer> fruitStorage;
+
+    public ReturnOperation(Map<String, Integer> fruitStorage) {
+        this.fruitStorage = fruitStorage;
+    }
+
+    @Override
+    public void apply(FruitTransaction transaction) {
+        fruitStorage.put(transaction.getFruit(), 
+                fruitStorage.getOrDefault(transaction.getFruit(), 0) + transaction.getQuantity());
+    }
+}
```

Comment:
Classes that are part of the strategy pattern should be placed in their own package for better organization. Consider moving this class to a `strategy` package.